reV/config/project_points.py

```diff
@@ -721,3 +724,160 @@ def split(cls, i0, i1, project_points):
                   curtailment=project_points.curtailment)
 
         return sub
+
+    @staticmethod
+    def _parse_lat_lons(lat_lons):
+        msg = ('Expecting a pair or multiple pairs of latitude and '
+               'longitude coordinates!')
+        if isinstance(lat_lons, str):
+            lat_lons = parse_table(lat_lons)
+            cols = [c for c in lat_lons if c.lower.startswith(('lat', 'lon'))]
+            lat_lons = lat_lons[sorted(cols)].values
+        elif isinstance(lat_lons, (list, tuple)):
+            lat_lons = np.array(lat_lons)
+        elif isinstance(lat_lons, (int, float)):
+            msg += ' Recieved a single coordinate value!'
+            logger.error(msg)
+            raise ValueError(msg)
+
+        if len(lat_lons.shape) == 1:
+            lat_lons = np.expand_dims(lat_lons, axis=0)
+
+        if lat_lons.shape[1] != 2:
+            msg += ' Received {} coordinate values!'.format(lat_lons.shape[1])
+            logger.error(msg)
+            raise ValueError(msg)
+
+        return lat_lons
+
+    @classmethod
+    def lat_lon_coords(cls, lat_lons, res_file, sam_config, tech=None,
+                       curtailment=None):
+        """
+        Generate ProjectPoints for gids nearest to given latitude longitudes
+
+        Parameters
+        ----------
+        lat_lons : str | tuple | list | ndarray
+            Pair or pairs of latitude longitude coordinates
+        res_file : str
+            Resource file, needed to fine nearest neighbors
+        sam_config : dict | str | list | SAMConfig
+            SAM input configuration ID(s) and file path(s). Keys are the SAM
+            config ID(s), top level value is the SAM path. Can also be a single
+            config file str. If it's a list, it is mapped to the sorted list
+            of unique configs requested by points csv. Can also be a
```

Comment:
I dont think this doc string is applicable to this class method. What would happen if you input a list for this? Same for the next class method